client/src/components/Onboarding/OnboardingUser.js

```diff
@@ -0,0 +1,259 @@
+      break;
+    case 'failed':
+      type = 'danger';
+      break;
+    case 'staffwait':
+    case 'userwait':
+      type = 'alert';
+      break;
+    case 'completed':
+      type = 'success';
+      break;
+    default:
+      type = 'normal';
+  }
+  if ('customStatus' in step) {
+    return <Pill type={type}>{step.customStatus}</Pill>;
+  }
+  switch (step.state) {
+    case 'pending':
+      return <Pill type={type}>Preparing</Pill>;
+    case 'staffwait':
+      return <Pill type="normal">Waiting for Staff Approval</Pill>;
+    case 'userwait':
+      return null;
+    case 'failed':
+      return <Pill type={type}>Unsuccessful</Pill>;
+    case 'completed':
+      return <Pill type={type}>Completed</Pill>;
+    case 'processing':
+      return (
+        <span className="onboarding-processing">
+          <Pill type={type}>Processing</Pill>
+          <LoadingSpinner placement="inline" />
+        </span>
+      );
+    default:
+      return <span>{step.state}</span>;
+  }
+}
+
+OnboardingStatus.propTypes = {
+  step: stepPropType.isRequired
+};
+
+OnboardingStatus.defaultProps = {};
+
+function OnboardingStep({ step }) {
+  return (
+    <div className="onboarding-step">
+      <div className="onboarding-step__name">{step.displayName}</div>
+      <div
+        /* eslint-disable react/no-danger */
+        className="onboarding-step__description"
+        dangerouslySetInnerHTML={{ __html: step.description }}
+      />
+      <div>
+        <OnboardingStatus step={step} />
+        <OnboardingActions step={step} />
+      </div>
+    </div>
+  );
+}
+
+OnboardingStep.propTypes = {
+  step: stepPropType.isRequired
+};
+
+OnboardingStep.defaultProps = {};
+
+function OnboardingUser() {
+  const { params } = useRouteMatch();
+  const dispatch = useDispatch();
+  const user = useSelector(state => state.onboarding.user);
+  const isStaff = useSelector(state =>
+    state.authenticatedUser.user ? state.authenticatedUser.user.isStaff : false
+  );
+  const loading = useSelector(state => state.onboarding.user.loading);
+  const error = useSelector(state => state.onboarding.user.error);
+
+  useEffect(() => {
+    dispatch({
+      type: 'FETCH_ONBOARDING_ADMIN_INDIVIDUAL_USER',
+      payload: {
+        user: params.username || ''
+      }
+    });
+    dispatch({
+      type: 'FETCH_AUTHENTICATED_USER'
+    });
+  }, [dispatch, params]);
+
+  if (loading) {
+    return (
+      <div data-testid="loading">
+        <LoadingSpinner />
+      </div>
+    );
+  }
+
+  if (error) {
+    return <div>Unable to retrieve your onboarding steps</div>;
+  }
+
+  return (
+    <div className="onboarding">
+      {isStaff ? (
+        <div className="onboarding__title">
+          Onboarding Administration for {user.username} - {user.lastName},{' '}
+          {user.firstName}
+        </div>
+      ) : (
+        <div className="onboarding__title">
+          The following steps must be completed before accessing the portal
+        </div>
```

Comment:
__Suggestion__: These texts are designed/styledstyled as if, and given a class name that suggests, the role of a heading. I propose:

```
<h2 className="onboarding__title">
          …
</h2>
```

The existing CSS code should already overwrite any unwanted, inherited styles (`margin`, `font-size`, etcetera).